Detections/MultipleDataSources/PotentialFodhelperUACBypassASIM.yaml

```diff
@@ -0,0 +1,38 @@
+id: ac9e233e-44d4-45eb-b522-6e47445f6582
+name: Potential Fodhelper UAC Bypass ASIM
+description: |
+  'This detection looks for the steps required to conduct a UAC bypass using Fodhelper.exe. By default this detection looks for the setting of the required registry keys and the invoking of the process within 1 hour - this can be tweaked as required.'
+severity: Medium
+requiredDataConnectors: []
+queryFrequency: 2h
+queryPeriod: 2h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1548.002
+query: |
+  imRegistry
+    | where EventType in ("RegistryValueSet", "RegistryKeyCreated")
+    | where RegistryKey has "Software\\Classes\\ms-settings\\shell\\open\\command"
+    | extend timekey = bin(TimeGenerated, 1h)
+    | join (imProcess
+    | where Process endswith "fodhelper.exe"
+    | where ParentProcessName endswith "cmd.exe" or ParentProcessName endswith "powershell.exe" or ParentProcessName endswith "powershell_ise.exe"
+    | extend timekey = bin(TimeGenerated, 1h)) on TimeKey, DvcId
```

Comment:
@petebryan : I suspect Dvc would be better here. DvcId tends to be specific to a source (Sysmon, MDE, SecurityEvent). Dvc is the best identified we have for the device, most often hostname, where you have better chance of correlating between different sources. 

What do you think?